### PR TITLE
Use single react 17 playground

### PR DESCRIPTION
**Pull request by epicfaace**
### Reasons for making this change

I tested putting all themes into a single react 17 playground and everything is working fine. Even though some of our dependencies like semantic-ui may technically not support react 17, since it works on the playground, I think we should just have one playground with react 17 for simplicity's sake.

This should simplify https://github.com/rjsf-team/react-jsonschema-form/pull/2605 hopefully.

### CODE REVIEW

File: packages/playground/src/app.js
Comment (epicfaace) [inline packages/playground/src/app.js:584]: apparently we don't actually need this, antd theme seems to work fine without it

File: package-lock.json
Comment (heath-freenome) [inline package-lock.json:4]: Consider reverting this file... or switching to node 16? I'm pretty sure the revert is perfectly fine.

Comment (epicfaace) [reply package-lock.json:4]: Because all our lockfiles currently have `"lockfileVersion": 1,` set, I'd like to keep this one using `"lockfileVersion": 1,` for consistency. We should upgrade all pacakges to node 16 maybe in another PR, though I've faced issues when trying to do so before, so it might be a more involved process to do so.

File: packages/playground/webpack.config.dist.js
Comment (heath-freenome) [inline packages/playground/webpack.config.dist.js:71]: I guess you removed less?

**Review by heath-freenome — APPROVED** [review]
Great job! Makes my PR better!

Would you be willing to update the `CHANGELOG.md` to reflect these changes? Then you can merge. Right now the `CHANGELOG.md` has this:
```
## Dev / docs / playground
- Added resolve aliases to `playground` so that the updated `material-ui` theme doesn't try to resolve the `@mui` libraries which require React 17 and fail to load as a result
- Added a new `playground-r17` for the `@rjsf/core` and `@rjsf/material-ui` version 4 and 5 themes on top of React 17
```

File: packages/playground/webpack.config.dist.js
Comment (epicfaace) [reply packages/playground/webpack.config.dist.js:71]: Yeah, it appears to still work without it, so I guess we're OK to just not have it anymore??

Comment (heath-freenome) [reply packages/playground/webpack.config.dist.js:71]: Does it cause problems if you leave it in?

File: packages/playground/src/app.js
Comment (heath-freenome) [reply packages/playground/src/app.js:584]: I wonder why that is? Maybe a recent upgrade to `antd` made the need for it irrelevant?  As long as the theme still works in the playground... ship it

**Comment by heath-freenome** [comment]
@epicfaace  Go ahead and merge this... and if you are happy, merge my PR as well!

File: packages/playground/src/app.js
Comment (epicfaace) [reply packages/playground/src/app.js:584]: yeah not sure haha... I'm happy to ship it because it's just the playground rather than an actual published npm package that people depend on.

**Comment by epicfaace** [comment]
I can't merge this, I don't have write access to this repo -- can you?

**Comment by heath-freenome** [comment]
@epicfaace done! Looking forward to having my PR merged!

**Pull request merged**
